bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/BigtableHBaseSettings.java

```diff
@@ -39,6 +39,14 @@ public abstract class BigtableHBaseSettings {
   private final String instanceId;
   private final int ttlSecondsForBackup;
 
+  public static final String BULK_MUTATION_CLOSE_TIMEOUT_SECONDS =
+      "bulk.mutation.close.timeout.seconds";
+
+  // Must be non-negative. Set to 0 to disable timeout.
+  private final long bulkMutationCloseTimeoutSeconds;
+
+  private static final long DEFAULT_BULK_MUTATION_CLOSE_TIMEOUT_SECONDS = 600;
```

Comment:
Do we always want to force close the BufferedMutator, even when it's not used in dataflow? If not, we should set the default to 0, and in CloudBigtableConfiguration override it to 10 miutes.

Reply:
You're right. Updated